### DISCUSSION TITLE
How to translate a part of the plugin that is not being translated

### DISCUSSION THREAD

**Discussion by karamzyatina** [plan-player-analytics/Plan#2573] [Q&A]
Good afternoon. I have translated almost the ENTIRE plugin site. Changed almost EVERY word. But here's the problem, using JarStingEditor, Using Notpad++ and other applications that help me in translation, cannot detect the translation for the following functions
![opera_iQXUSv4QUQ](https://user-images.githubusercontent.com/67735224/187072309-ce0adfca-5a13-46d9-a824-63b1072adfa6.png)
![opera_VHwUOPaQyC](https://user-images.githubusercontent.com/67735224/187072312-72ee6bc1-df72-4d93-a77b-585c085265b5.png)
![opera_1MTIRHJBpJ](https://user-images.githubusercontent.com/67735224/187072314-02fa2d0e-7a2b-4f6f-b96e-3435ce71f464.png)
![opera_6BtsIGjSei](https://user-images.githubusercontent.com/67735224/187072317-d4d1f724-3a95-4971-ac9a-9112d7664d24.png)
![opera_xppNyjfZEJ](https://user-images.githubusercontent.com/67735224/187072340-71aa814b-9185-4936-a82b-fc274e7cc944.png)

Perhaps these are not all the words that I can't translate, there may still be many of them, but we need to carefully check every corner of the site

**Comment by AuroraLS3** [comment]
https://github.com/plan-player-analytics/Plan/wiki/Localization

**Comment by AuroraLS3** [comment]
Date formatting is done in the browser by Javascript and data from plugins does not yet support localization - This will hopefully be possible to address once React rewrite completes.

**Comment by AuroraLS3** [comment]
Build 1876 has the server page rewritten in React (If you enable Frontend_BETA from config) https://github.com/plan-player-analytics/Plan/actions/runs/2942154566

**Reply by karamzyatina** [reply]
Well, thank you. You've been very helpful. How can I remove the inscription 5.5 build 1876, or replace it with another one
![opera_ILIlbVPTSj](https://user-images.githubusercontent.com/67735224/187087573-1cc9453f-c835-40b1-9801-d2f670e0f113.png)
?

**Reply by AuroraLS3** [reply]
It is currently not possible since html customization is not yet supported in the beta.

**Comment by karamzyatina** [comment]
And the month doesn't change. I changed all the parameters of the names of months, days and nothing helped. The meaning is in English anyway, maybe you will add it in the future.
![opera_RAIvWIN9Cb](https://user-images.githubusercontent.com/67735224/187088396-48745afc-b7cc-4a0b-b9c2-705e9a57e803.png)
